### ISSUE TITLE
News: Federate ALT Text

### ISSUE THREAD

**Issue by benpate** [EmissarySocial/bandwagon#227]
It's possible to add alt text for the news banner and icon, but this information is not translated into ActivityPub.

Alt text should be stored directly with the attachment, and not in a custom field, which will make it much easier for Emissary to include it natively (without using SocialRules)
